hydra-node/test/Hydra/API/RestServerSpec.hs

```diff
@@ -17,3 +21,27 @@ spec = parallel $ do
 
   roundtripAndGoldenSpecs
     (Proxy @(ReasonablySized (DraftCommitTxRequest Tx)))
+
+  aroundAll withJsonSpecifications $
+    -- XXX: It's unintuitive that dir/api.yaml is in fact a JSON file holding
+    -- the JSON schema!
+    specify "Validate /commit publish api schema" $ \dir -> do
+      property $
+        withMaxSuccess 1 $ do
+          conjoin
+            [ prop_validateJSONSchema @(DraftCommitTxRequest Tx) (dir </> "api.yaml") (key "channels" . key "/commit" . key "publish" . key "message" . key "payload")
+            , prop_specIsComplete @(ReasonablySized (DraftCommitTxRequest Tx)) (dir </> "api.yaml") apiSpecificationSelector
```

Comment:
This spec is pointless as the request has only a single instance and there is no need to check for completeness.